src/components/HighlightedText/readme.md

```diff
@@ -0,0 +1,85 @@
+##### HighlightedText base
+
+```js
+import React, { useState } from 'react';
+import {  HighlightedText } from 'react-rainbow-components';
+
+const style = {
+    fontSize:"16px",
+    maxWidth:"700px",
+    textAlign:"center",
+    padding:"20px",
+    margin:"auto",
+};
+
+const parts = [
+        {
+          "value" : "Apples come in several ",
+          "type" : "text"
+        },
+        {
+          "value" : "varieties",
+          "type" : "hit"
+        },
+        {
+          "value" : ", including Fuji, Granny Smith, and Honeycrisp.",
+          "type" : "text"
+        }
+];
+
+<HighlightedText parts={parts} style={style}  />
+
+```
+
+##### HighlightedText with custom styles
+
+```js
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import {  HighlightedText } from 'react-rainbow-components';
+
+const style = {
+    fontSize:"16px",
+    maxWidth:"700px",
+    textAlign:"center",
+    padding:"20px",
+    margin:"auto",
+};
+
+const parts = [
+        {
+          "value" : "Apples come in several ",
+          "type" : "text"
+        },
+        {
+          "value" : "varieties",
+          "type" : "hit"
+        },
+        {
+          "value" : ", including Fuji, Granny Smith, and Honeycrisp.",
+          "type" : "text"
+        }
+];
+
+const TextContainer = styled.span`
+    color: grey;
+`;
+
+const textComponent = ({children}) => {
```

Comment:
I think that if we have a simple example, we don't need that, we can pass directly the `TextContainer` to the `textComponent` like `textComponent={textContainer}`